modules/morseUtils.js

```diff
@@ -0,0 +1,24 @@
+import { code } from "../modules/array.js";
+
+export const convertToMorse = (str) => {
+  return str
+    .toUpperCase()
+    .split("")
+    .map((el) => {
+      if (code[el]) {
+        return code[el];
+      }
+      return el;
```

Comment:
Could replace this if with a lazy evaluation
`=> {
return code[el] || el;
}`

if code[el] is true it gets returned if not el is returned

Reply:
thankyou, I've done this